contrib/aws/aws-sdk-go-v2/aws/aws.go

```diff
@@ -101,6 +126,228 @@ func (mw *traceMiddleware) startTraceMiddleware(stack *middleware.Stack) error {
 	}), middleware.After)
 }
 
+func extractResourceNameFromParams(requestInput middleware.InitializeInput, awsService string) (string, string, error) {
+	var resourceNameKey, resourceNameValue string
```

Comment:
```suggestion
	var k, v string
```
I think it's clear from the name of the function that this is going to be a resource name/value, so just k,v should suffice here.